doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -71,18 +71,29 @@ It must be called before each call to X509_verify_cert(), i.e. a I<ctx> is only
 good for one call to X509_verify_cert(); if you want to verify a second
 certificate with the same I<ctx> then you must call X509_STORE_CTX_cleanup()
 and then X509_STORE_CTX_init() again before the second call to
-X509_verify_cert(). The trusted certificate store is set to I<store>, the end
-entity certificate to be verified is set to I<x509> and a set of additional
-certificates (which will be untrusted but may be used to build the chain) in
-I<chain>. Any or all of the I<store>, I<x509> and I<chain> parameters can be
+X509_verify_cert().
+The trusted certificate store is set to I<trust_store> of type B<X509_STORE>.
+This may be NULL because there are no trusted certificates or because
+they are provided simply as a list using X509_STORE_CTX_set0_trusted_stack().
+The end entity certificate to be verified is set to I<target>,
+and a set of additional certificates may be provided in I<untrusted>,
+which will be untrusted but may be used to build the chain.
+Any or all of the I<trust_store>, I<target> and I<untrusted> parameters can be
 B<NULL>.
 
 X509_STORE_CTX_set0_trusted_stack() sets the set of trusted certificates of
 I<ctx> to I<sk>. This is an alternative way of specifying trusted certificates
-instead of using an B<X509_STORE>.
+instead of using an B<X509_STORE> where its complexity is not needed.
```

Comment:
This is not primarily about "complexity" avoidance.  Sometimes one actually wants to *ensure* that only the particular certificates in question are used to build the chain, separately from any preconfigured "store".

Reply:
Good point. 
That addition was inspired by a similar comment in the code.
I've just extended this by
> or to make sure that only the given set `I<sk>` of certificates are trusted.